src/magma/vk/ComputePipeline.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file ComputePipeline.cpp
+ * @brief Class create compute pipeline
+ * @version 0.1
+ * @date 2021-02-09
+ */
+#include "magma/vk/ComputePipeline.h"
+
+#include "magma/vk/vulkan_common.h"
+
+ComputePipelineInfo::ComputePipelineInfo() {
+    _pipelineLayoutInfo = {};
+    _pipelineLayoutInfo.sType = VK_STRUCTURE_TYPE_PIPELINE_LAYOUT_CREATE_INFO;
+    _pipelineLayoutInfo.setLayoutCount = 0;
+    _pipelineLayoutInfo.pSetLayouts = nullptr;
+    _pipelineLayoutInfo.pushConstantRangeCount = 0;
+    _pipelineLayoutInfo.pPushConstantRanges = nullptr;
+}
+
+void ComputePipelineInfo::setLayout(const std::vector<VkDescriptorSetLayout> &descriptorSetLayouts) {
```

Comment:
In GraphicsPipeline this method is called ```setLayouts``` (with 's' at the end).
So there were no overloaded methods.

Reply:
resolve after discussion